doc/user/content/integrations/monitor/datadog.md

```diff
@@ -0,0 +1,165 @@
+---
+title: "Monitor Materialize with Datadog"
+description: "Use Datadog to monitor Materialize."
+menu:
+  main:
+    parent: ""
+    name: 
+---
```

Comment:
Not for the sake of aliases but to have an access to the article inside the live Preview (unless there is an alternative way that I'm unaware of. In that case, sorry!)
```suggestion
aliases:
  /monitoring/datadog
---
```